scripts/SREP/update-upgrade-policy-state/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+file: script.sh
+name: update-upgrade-policy-state
+description: Cancel the upgrade policy of the current OpenShift cluster.
+author: sbai
+allowedGroups: 
```

Comment:
I do not think we would want this to be executed by the other team than SREP?

Reply:
If this script will only be used by our SREP, I can remove other teams. Actually, I can see most of the scripts in the SREP folder included CEE as allowedGroup.

Reply:
let's keep this for SREP only for now, if other teams will use this as well we can add them later